### DISCUSSION TITLE
`displayUsingLabels` as default

### DISCUSSION THREAD

**Discussion by mrleblanc101** [laravel/nova-issues#4995] [Ideas & Feature Requests] [answered]
Hi,
Maybe I'm missing something, but why do we need to specify `->displayUsingLabels()` for every `Select::make`.
Shouldn't it be the default ? Why would you display the value of an Enum for exemple.
What's the point of defining Labels if you don't use them.

**Comment by crynobone** [comment]
It something we can only consider for Nova 5.

**Reply by mrleblanc101** [reply]
Ok, but that's not really an answer. 
Is it being considered to be the new default in Nova 5 ? 
Or I am just mistaken and the current behavior is correct.
When would Nova 5 launch approximately ? 
I think Nova switched to a 1 year cycle, so around April 2023 ?

**Reply by crynobone** [reply]
We don't introduce breaking changes to a public API during minor or patch release. 

Also, 1 year release cycle is just for Laravel Framework, Nova doesn't have to follow it unless new version introduce or remove feature required by Nova and force us to release a new major version.

**Reply by mrleblanc101** [reply]
But I've never said it should be released in a minor or a patch 🤔 
I'm simply asking if it's being considered to change the behavior.
Or if it will stay like this indefinitely.

Thanks for the clarification of the release schedule.
So no ETA on Nova 5 I suppose ?

**Comment by crynobone** [comment]
> But I've never said it should be released in a minor or a patch 🤔
I'm simply asking if it's being considered to change the behavior.
Or if it will stay like this indefinitely.

We haven't discussed and decided on it yet. Both options have their own merit.

If you want this to be the default you can always create a class and configure the default. e.g: `App\Nova\Fields\Select`.

**Comment by davidhemphill** [comment, marked as answer]
It works this way because users can store literal strings in a database column without the overhead of a key/value concept. Probably not going to make a BC change even in the next version, as we tend to avoid them if we can.
